integration-tests/test_gradle_plugin.py

```diff
@@ -585,6 +585,7 @@ def __init__(self, *args, **kwargs):
     def setUpClass(cls):
         super().setUpClass()
 
+    @util.skip_on_windows("ujson installation broken on Windows")
```

Comment:
Adding a Jira would be nice so that we know when to unskip this

Reply:
I'll add it in next PR